cmd/kubexit/main.go

```diff
@@ -47,74 +45,74 @@ func main() {
 		graveyard = strings.TrimRight(graveyard, "/")
 		graveyard = filepath.Clean(graveyard)
 	}
-	log.Printf("Graveyard: %s\n", graveyard)
+	log.Info("Graveyard:", "graveyard", graveyard)
 
 	ts := &tombstone.Tombstone{
 		Graveyard: graveyard,
 		Name:      name,
 	}
-	log.Printf("Tombstone: %s\n", ts.Path())
+	log.Info("Tombstone:", "tombstone", ts.Path())
 
 	birthDepsStr := os.Getenv("KUBEXIT_BIRTH_DEPS")
 	var birthDeps []string
 	if birthDepsStr == "" {
-		log.Println("Birth Deps: N/A")
+		log.Info("Birth Deps: N/A")
 	} else {
 		birthDeps = strings.Split(birthDepsStr, ",")
-		log.Printf("Birth Deps: %s\n", strings.Join(birthDeps, ","))
+		log.Info("Birth Deps:", "birth deps", strings.Join(birthDeps, ","))
 	}
 
 	deathDepsStr := os.Getenv("KUBEXIT_DEATH_DEPS")
 	var deathDeps []string
 	if deathDepsStr == "" {
-		log.Println("Death Deps: N/A")
+		log.Info("Death Deps: N/A")
 	} else {
 		deathDeps = strings.Split(deathDepsStr, ",")
-		log.Printf("Death Deps: %s\n", strings.Join(deathDeps, ","))
+		log.Info("Death Deps:", "death deps", strings.Join(deathDeps, ","))
 	}
 
 	birthTimeout := 30 * time.Second
 	birthTimeoutStr := os.Getenv("KUBEXIT_BIRTH_TIMEOUT")
 	if birthTimeoutStr != "" {
 		birthTimeout, err = time.ParseDuration(birthTimeoutStr)
 		if err != nil {
-			log.Printf("Error: failed to parse birth timeout: %v\n", err)
+			log.Error(err, "Error: failed to parse birth timeout")
 			os.Exit(2)
 		}
 	}
-	log.Printf("Birth Timeout: %s\n", birthTimeout)
+	log.Info("Birth Timeout:", "birth timeout", birthTimeout)
 
 	gracePeriod := 30 * time.Second
 	gracePeriodStr := os.Getenv("KUBEXIT_GRACE_PERIOD")
 	if gracePeriodStr != "" {
 		gracePeriod, err = time.ParseDuration(gracePeriodStr)
 		if err != nil {
-			log.Printf("Error: failed to parse grace period: %v\n", err)
+			log.Error(err, "Error: failed to parse grace period")
 			os.Exit(2)
 		}
 	}
-	log.Printf("Grace Period: %s\n", gracePeriod)
+	log.Info("Grace Period:", "grace period", gracePeriod)
 
 	podName := os.Getenv("KUBEXIT_POD_NAME")
 	if podName == "" {
 		if len(birthDeps) > 0 {
-			log.Println("Error: missing env var: KUBEXIT_POD_NAME")
+			log.Error(errors.New("missing env var: KUBEXIT_POD_NAME"), "Error: missing env var: KUBEXIT_POD_NAME")
```

Comment:
```suggestion
			log.Error(errors.New("missing env var: KUBEXIT_POD_NAME"), "missing env var", "var_name", "KUBEXIT_POD_NAME)
```

structured logging allows you to easily filter when searching in logs